apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSource.java

```diff
@@ -103,6 +103,52 @@ public StandardGrpcRdaSource(
     this.minIdleMillisBeforeConnectionDrop = minIdleMillisBeforeConnectionDrop;
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * <p>This implementation reads starting sequence number from the database, gets the API version
+   * number from RDA API, then downloads several claims from the API.
+   *
+   * @param sink to process batches of objects
+   * @return true if all actions were completed successfully
+   * @throws Exception if any of the actions threw an exception
+   */
+  @Override
+  public boolean performSmokeTest(RdaSink<TMessage, TClaim> sink) throws Exception {
+    boolean successful = false;
+    log.info("smoke test: starting");
+
+    // Query the database to get a starting sequence number.  This verifies that the database is
+    // accessible.
+    final long startingSequenceNumber =
+        sink.readMaxExistingSequenceNumber().orElse(MIN_SEQUENCE_NUM);
```

Comment:
If we start from the rda_api_progress table, there's a chance that when we deploy there is no more data for us to consume and a connection to the RDA API would just be waiting for more data to come in.  Am I reasoning this correctly?

Reply:
Yes.  If we pass the value from the `rda_api_progress` table to the RDA API when we start downloading claims then we might have to wait if there are no newer claims available.  That's why we start from`MIN_SEQUENCE_NUM` when we actually start retrieving claims in the next step.

Reply:
oh I see, its just for logging, my mistake